SPMD and oneAPI DBSCAN docs (#1979)
oneapi-src/oneDAL: docs/source/includes/clustering/dbscan-introduction.rst

.. ******************************************************************************
.. * Copyright 2021 Intel Corporation
.. *
.. * Licensed under the Apache License, Version 2.0 (the "License");
.. * you may not use this file except in compliance with the License.
.. * You may obtain a copy of the License at
.. *
.. *     http://www.apache.org/licenses/LICENSE-2.0
.. *
.. * Unless required by applicable law or agreed to in writing, software
.. * distributed under the License is distributed on an "AS IS" BASIS,
.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
.. * See the License for the specific language governing permissions and
.. * limitations under the License.
.. *******************************************************************************/

> The DBSCAN algorithm solves :capterm:`clustering` problem by the following way:

Reviewer: ```suggestion
The DBSCAN algorithm solves :capterm:`clustering` problem in a following way:
```

> given a set of observations, it groups together ones that are closely packed

Reviewer: ```suggestion
given a set of observations, it groups together the ones that are closely packed
```

> (having at least :math:`min_observations` number of neighbors inside distance :math:`epsilon`)

Reviewer: ```suggestion
(having at least ``min_observations`` neighbors within the :math:`\varepsilon` distance)
```

> and marking other ones as outliers.

Reviewer: ```suggestion
and marking the other ones as outliers.
```

> 

Reviewer: I'd suggest we work on this description offline because it's not the best one

Author: Changed
